viewer/packages/utilities/src/three/VariableWidthLine.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { Color, Vector3 } from 'three';
+import { Line2, LineGeometry, LineMaterial } from 'three-stdlib';
+
+export class VariableWidthLine {
```

Comment:
For me, this name implies that you can set line width in runtime but I guess it can be added in the future if we will have some exposed styling adjustment for annotations.

Reply:
True :sweat_smile: I'll see if I can come up with a better name :thinking: